.release.cloudbuild.yaml

```diff
@@ -243,14 +243,14 @@ steps:
     docker push gcr.io/ml-pipeline/google/pipelines-test/visualizationserver:$(cat /workspace/mm.ver)
 
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['pull', 'gcr.io/tfx-oss-public/ml_metadata_store_server:v0.21.1']
```

Comment:
https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L246

Reply:
Can you explain a bit more? A little bit of confused by the link

Reply:
I means this part also need changes

Reply:
I guess you meant https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L263 ?

Reply:
Changed accordingly